app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -77,6 +74,14 @@ class ConfigurationHeuristic(private val heuristicConfigurationData: HeuristicCo
       new HeuristicResultDetails(
```

Comment:
Add recommendations to the heuristic details. If multiple of them fails, we take the max and user has to fix one to know there is more to fix.